src/libraries/angular.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (routes) => {
+  return `
+  angular.module('foundation.dynamicRouting').config([
```

Comment:
`foundation.dynamicRouting` can be renamed to `dynamicRouting` and `FoundationStateProvider` can be renamed to `BaseAppsStateProvider`.  These changes are available as part of [angular-dynamic-routing](https://github.com/base-apps/angular-dynamic-routing) v1.1